UI/src/view/linear_asset/speedLimitForm.js

```diff
@@ -132,7 +132,7 @@
   var bindEvents = function(selectedSpeedLimit, feedbackCollection, backend) {
     new FeedbackDataTool(feedbackCollection, 'speedLimit', authorizationPolicy);
 
-    if (authorizationPolicy.userRoles.length === 0){
+    if (!authorizationPolicy.fetched){
```

Comment:
This code need to be re-think, Doesn't seem correct start call the getUserRoles in differents part of application. Could you investigate if it possible use the eventbus.on(roles:fecthed) and remove this?